CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# v2.3.0
+
+This is a patch version change keep this projects versioning coordinated with [Reaction Platform](https://github.com/reactioncommerce/reaction-platform), [Reaction](https://github.com/reactioncommerce/reaction), and our [Example Storefront](https://github.com/reactioncommerce/example-storefront).
```

Comment:
This isn't a patch, it's a `minor` version

Reply:
updated